data/src/main/java/org/akvo/flow/data/entity/form/FormMapper.kt

```diff
@@ -25,6 +25,42 @@ import javax.inject.Inject
 
 class FormMapper @Inject constructor() {
 
+    fun mapForms(cursor: Cursor?): List<DataForm> {
+        val forms = mutableListOf<DataForm>()
+        if (cursor != null && cursor.moveToFirst()) {
+            do {
+                val id = getIntColumnValue(cursor, SurveyColumns._ID)
+                val formId = getStringColumnValue(cursor, SurveyColumns.SURVEY_ID)
+                val surveyId = getIntColumnValue(cursor, SurveyColumns.SURVEY_GROUP_ID)
+                val formVersion = getStringColumnValue(cursor, SurveyColumns.VERSION)
+                val name = getStringColumnValue(cursor, SurveyColumns.NAME)
+                val type = getStringColumnValue(cursor, SurveyColumns.NAME)
+                val location = getStringColumnValue(cursor, SurveyColumns.NAME)
+                val filename = getStringColumnValue(cursor, SurveyColumns.NAME)
+                val language = getStringColumnValue(cursor, SurveyColumns.NAME)
```

Comment:
I am  not sure about inner workings but here its ok for the second parameter to be `SurveyColumns.NAME`?

Reply:
oh no, that was an error I had copied from another part, will fix it